src/main/kotlin/org/vividus/testsite/api/PageController.kt

```diff
@@ -0,0 +1,40 @@
+package org.vividus.testsite.api
+
+import java.io.File
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.core.io.Resource
+import org.springframework.core.io.ResourceLoader
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.ui.Model
+import org.springframework.ui.set
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+
+@Controller()
+class PageController @Autowired constructor(
+    private val resourceLoader: ResourceLoader
```

Comment:
can be injected into getImageAsResponseEntity

Reply:
what's the profit?